### ISSUE TITLE
[Bug Report] v-data-table header WCAG AA violated

### ISSUE THREAD

**Issue by keigoleung** [vuetifyjs/vuetify#5017]
### Versions and Environment
Vuetify: 
Vue: 1.2.1
Browsers: any
OS: any

### Steps to reproduce
Use HTML_CodeSniffer to check the data-table page.
error was "The relationship between td elements and their associated th elements is not defined. As this table has multiple levels of th elements, you must use the headers attribute on td elements."
relate to this https://www.w3.org/TR/WCAG20/#content-structure-separation-programmatic
the issue was believed to be a <tr> with classname v-datatable__progress
because table has multiple level th, td will need to assign with a headers attribute.
but also header is unable to set <th> "id" attribute.

### Expected Behavior
pass WCAG validate

### Actual Behavior
doesn't pass WCAG validate

**Comment by deadbeef404** [comment]
Link to HTML_CodeSniffer: https://squizlabs.github.io/HTML_CodeSniffer/

**Comment by johnleider** [comment]
Please follow the <a href="https://github.com/vuetifyjs/vuetify/blob/master/.github/CONTRIBUTING.md">guidelines</a> on how to report an issue. In particular, provide an example on <a href="https://jsfiddle.net/">www.jsfiddle.net</a> (or a similar service) that reproduces the problem. If necessary, create a repository for us to clone with a minimal reproduction. repositories of actual projects will generally not be accepted .

Thank you.
